src/ansys/units/unit_registry.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import yaml
+
+import ansys.units as ansunits
+
+
+class UnitRegistry:
+    """
+    Initializes a Unit Registry.
+
+    Parameters
+    ----------
+    Config: filename.yaml
+        Custom units.
+    Other: Dict
+        Dictionary of units.
+    Methods
```

Comment:
Methods is blank?  I guess you can document the methods individually below?

Reply:
Since there is only a __str__ method. Should it be removed for the time being?

Reply:
Yes, that seems reasonable.